bundles/pixi.js-node/README.md

```diff
@@ -0,0 +1,98 @@
+<div align="center">
+  <h1>@pixi/node</h1>
+</div>
+<br>
+
+**We are now a part of the [Open Collective](https://opencollective.com/pixijs) and with your support you can help us make PixiJS even better. To make a donation, simply click the button below and we'll love you forever!**
+
+<div align="center">
+  <a href="https://opencollective.com/pixijs/donate" target="_blank">
+    <img src="https://opencollective.com/pixijs/donate/button@2x.png?color=blue" width=250 />
+  </a>
+</div>
+
+## Setup
+
+### Install
+
+```
+npm install @pixi/node
+```
+
+There is no default export. The correct way to import PixiJS is:
+
+```js
+import * as PIXI from "@pixi/node";
+```
+
+### Error installing canvas package
+
+The [canvas](https://www.npmjs.com/package/canvas) library currently being used does not have a pre-built version for every environment.
+When the package detects an unsupported environment, it will try to build from source.
+
+To build from source you will need to make sure you have the following dependencies installed and then reinstall:
+
+`brew install pkg-config cairo pango libpng jpeg giflib librsvg`
+
+## Basic Usage Example
+This example uses `express`, however, you can use any framework you like.
+
+```js
+import express from 'express';
+import * as PIXI from '@pixi/node';
+
+const app = express();
+const port = 3000;
+
+app.use(express.static(__dirname))
+app.listen(port, () => {
+  console.log(`Example app listening on port ${port}`)
+})
+
+// This package requires the new asset loader to be used.
+// Initialize the new assets loader
+await PIXI.Assets.init({
+    basePath: 'http://localhost:3000/'
```

Comment:
Do absolute file paths not work?

Reply:
I think the loader expects the filename to be a URL. When loading, it tries to fetch the resource.
That's why I added #8470 (which I noticed isn't fully complete yet, because we need to add a ResourceLoader to INSTALLED, is that correct? Otherwise I am getting `Unrecognized source type to auto-detect Resource`)